build.sbt

```diff
@@ -88,7 +88,7 @@ libraryDependencies ++= Seq(
   "org.julienrf" %% "play-json-derived-codecs" % "6.0.0",
   "io.cucumber" % "gherkin" % "5.2.0",
   "com.typesafe.play" %% "anorm" % "2.5.3",
-  "mysql" % "mysql-connector-java" % "8.0.19",
+  "mysql" % "mysql-connector-java" % "5.1.48",
```

Comment:
I don't know if we want to include this "rollback" in the Github source code :man_shrugging: